string type
geometry_msgs/Point geometry
# z coordinate will be stripped
